empty_predicate.go

```diff
@@ -5,12 +5,18 @@ import (
 	"os"
 )
 
+var (
+	ErrorEmptyPredStat    = fmt.Errorf("emptyPredicate: lstat")
```

Comment:
Please use `errors.New("msg")` when the error content is static (i.e. there are no variables here).